pkg/client/kubernetes/client.go

```diff
@@ -44,6 +49,16 @@ var (
 // KubeConfig is the key to the kubeconfig
 const KubeConfig = "kubeconfig"
 
+func init() {
+	// enable protobuf for Gardener API for controller-runtime clients
+	protobufSchemeBuilder := runtime.NewSchemeBuilder(
+		gardenercorescheme.AddToScheme,
+		settingsscheme.AddToScheme,
```

Comment:
What about the new seedmanagement scheme?

Reply:
Good point, thanks :)
Added.
/squash